css/style.css

```diff
@@ -0,0 +1,48 @@
+body{
+    margin: 0;
+    background-color: rgba(234, 225, 201, 1);
+    font-family: "Jaini Purva";
```

Comment:
Use a full font stack.

Reply:
missing some other basic default rules including line-height, font-weight, font-style, color, etc. that should be set by default.